codebase/src/cpp/ieee1516e/src/portico/types/FixedRecordTypeImpl.cpp

```diff
@@ -0,0 +1,32 @@
+#include "RTI\portico\types\FixedRecordType.h"
+
+FixedRecordType::FixedRecordType(const std::string& name, std::list<Field> fields)
+{
+    this->name = name;
+    this->fields = fields;
+}
+
+FixedRecordType::~FixedRecordType()
+{
+
+}
+
+std::list<Field>& FixedRecordType::getFields()
+{
+    return this->fields;
+}
+
+bool FixedRecordType::operator==(const FixedRecordType& other)
+{
+    return this->name == other.name /*&& this->fields == other.fields*/;
```

Comment:
My instinct tells me this should not be commented out?

Reply:
fixed. took me a bit but got it sorted. Totally on my own.  No help from anyone.